ui/core/proto_utils/logs_parser.tsx

```diff
@@ -371,35 +371,44 @@ export class DamageDealtLog extends SimLog {
 		const spellSchoolString = typeof this.spellSchool === 'number' ? spellSchoolNames.get(this.spellSchool) : undefined;
 		return (
 			<>
-				{this.isHealing() ? `Heal ` : ''}
-				{this.isShielding() ? `Shield ` : ''}
-				{this.miss
-					? 'Miss'
-					: this.dodge
-					? 'Dodge'
-					: this.parry
-					? 'Parry'
-					: this.glance
-					? 'Glance'
-					: this.block
-					? this.crit
-						? 'Critical Block'
-						: 'Block'
-					: this.crit
-					? 'Crit'
-					: this.crush
-					? 'Crush'
-					: this.tick
-					? 'Tick'
-					: 'Hit'}
+				{this.isHealing() ? `Healed ` : ''}
+				{this.isShielding() ? `Shielded ` : ''}
+				{!(this.isHealing() || this.isShielding()) && (
+					<>
+						{this.miss
+							? 'Miss'
+							: this.dodge
+							? 'Dodge'
+							: this.parry
+							? 'Parry'
+							: this.glance
+							? 'Glance'
+							: this.block
+							? this.crit
+								? 'Critical Block'
+								: 'Block'
+							: this.crit
+							? 'Crit'
+							: this.crush
+							? 'Crush'
+							: this.tick
+							? 'Tick'
+							: 'Hit'}
+					</>
+				)}
 				{` `}
 				{this.target?.toHTML() || ''}
 				{!this.miss && !this.dodge && !this.parry ? (
 					<>
 						for{' '}
-						<strong className={clsx('text-danger', spellSchoolString && `spell-school-${spellSchoolString.toLowerCase()}`)}>
-							{this.amount.toFixed(2)} damage{spellSchoolString && <> ({spellSchoolString})</>}
-						</strong>
+						{this.isHealing() ? (
```

Comment:
Could you add ` || this.isShielding()` here? If you don't hide logs shieldings still show up as damage otherwise:
<img width="829" alt="image" src="https://github.com/user-attachments/assets/847efde5-e880-4059-893b-ef19c5a80178">

This looks nicer:
<img width="821" alt="image" src="https://github.com/user-attachments/assets/7b53be14-a332-4f07-934f-1e606356cde4">

Reply:
Oops, nice catch!